src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+        # Available parameters
+        self.parameters_map = {
+            ParameterType.PM25: ('pm2p5', MeasurementUnit.MICROGRAMS_PER_CUBIC_METER),
+            ParameterType.PM10: ('pm10', MeasurementUnit.MICROGRAMS_PER_CUBIC_METER),
+            ParameterType.NO2: ('no2', MeasurementUnit.MICROGRAMS_PER_CUBIC_METER),
+            ParameterType.SO2: ('so2', MeasurementUnit.MICROGRAMS_PER_CUBIC_METER),
+            ParameterType.CO: ('co', MeasurementUnit.PARTS_PER_MILLION),
+            ParameterType.O3: ('o3', MeasurementUnit.MICROGRAMS_PER_CUBIC_METER),
+        }
+        
+        # Cache for loaded datasets
+        self._dataset_cache: Dict[str, xr.Dataset] = {}
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)